hw_lesson19.py

```diff
@@ -0,0 +1,85 @@
+from unittest import mock
+from nose.tools import *
+from Lesson11 import func_for_task2
+
+"""
+Task 1
+
+Pick your solution to one of the exercises in this module.
+Design tests for this solution and write tests using unittest library.
+"""
+
+import unittest
+from Lesson12 import first_func, second_func
+
+
+class Task1TestCase(unittest.TestCase):
+
+    def test_Lesson12_first_func(self):
+        full = first_func()
+        self.assertEqual(full, print('This is first function'))
```

Comment:
!!

`print` function returns `None`.
`assertEqual` work the same to `assert 1 + 2 == 3`

In your case, first_func also returns nothing. And only because of that, this test being passed.
But here comes the problem: we can't test such functions. We can't know what's being made inside (I mean prints etc). I would suggest to make your function to return somethingand then test it:
```
def example_sum(x, y):
    return x + y

...

def test_example_sum():
    self.assertEqual(example_sum(1, 2), 3)
```

Reply:
Changed